roles/README.adoc

```diff
@@ -359,7 +371,25 @@ See https://docs.ansible.com/ansible/latest/dev_guide/overview_architecture.html
 ====
 * Use fully qualified role names in examples, like: `linux-system-roles.$ROLENAME` (with the Galaxy prefix).
 * Use RFC https://tools.ietf.org/html/rfc5737[5737], https://tools.ietf.org/html/rfc7042#section-2.1.1[7042] and https://tools.ietf.org/html/rfc3849[3849] addresses in examples.
-* Modules should have complete metadata, documentation, example and return blocks as described in the https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html[Ansible docs].
+====
+
+=== README file
+[%collapsible]
+====
+Rationale::
+The documentation is essential for the success of the content.
+Place the README file in the root directory of the role.
+The README file exists to introduce the user to the purpose of the role and any important information on how to use it, such as credentials that are required.
+
+At minimum include:
+* Example playbooks that allow users to understand how the developed content works are also part of the documentation.
+* The inbound and outbound role argument specifications
+* List of user-facing capabilities within the role
+* The unit of automation the role provided
```

Comment:
I'm not sure if I get what a "unit of automation is", could you please expand this a bit?

```suggestion
* The unit of automation the role provides
```